docs/index.md

```diff
@@ -16,7 +16,10 @@ hide:
 <div class="bitovi-row">
     <div class="bitovi-column">
         <h1>BitOps centralizes, organizes, and deploys your Infrastructure-as-Code</h1>
-        <a class="md-button md-button--primary" href="getting-started">Show me how</a>
+        <div>
+        <a class="md-button md-button--primary" href="getting-started">Show me the doc</a>
+        <a class="md-button md-button--primary" href="https://youtu.be/BiytYu3EefY">Show me a video</a>
```

Comment:
```suggestion
        <a class="md-button md-button--primary" href="getting-started">Show me the doc</a>
```
One CTA button is enough.

Instead of "show me the video" button, the idea was to replace the big logo block and adding a video iframe of a similar size. Big BitOps logo is not informational and just takes a precious space.
![image](https://user-images.githubusercontent.com/1533818/212999532-9b9e51c6-7bf8-4e02-bbe4-4010a7e53149.png)

Additionally, bitops logo image might need a rework to make it compact and inline `<image> BitOps`.


I can work on that, if you think it becomes too involved. Just let me know, can do it after a landing page.